src/cs/tool/Program.cs

```diff
@@ -32,6 +32,7 @@ static unsafe void Main(string[] args)
             }
 
             var ApiTable = MsQuic.Open();
+            const string DomainName = "google.com";
```

Comment:
But you still hard code it here...? How does this actually fix it? My understanding of the problem is that you any URL in here.

Reply:
What does this "tool" want to do?
Connection establishment over internet using C#?
Then any endpoint is needed, isn't it?

Reply:
Yes, just a sanity check that it can use C# + msquic to make a connection.

Reply:
What is the next step?
- Report ignoring?
- Remove this tool?
- Connect localhost?
- Is there any acceptable endpoint?

Reply:
Let's just remove the tool.

Reply:
Crap. Looks like the tests were using this. Let's add back the tool, make the URL a command line arg, and then update the test code to pass in google.com instead.